hello-world-ux-aspects/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
On second thought, it might be better to remove the package-lock.json file as we would have to keep going into this project and updating the dependencies any time we release a new version of ux-aspects.